awx_collection/plugins/inventory/tower.py

```diff
@@ -98,15 +98,15 @@
 # ansible-inventory -i @tower_inventory --list
 '''
 
-import re
 import os
-import json
+import re
 from ansible.module_utils import six
-from ansible.module_utils.urls import Request, urllib_error, ConnectionError, socket, httplib
+from ansible.module_utils.urls import Request
```

Comment:
It's a little awkward to move the other things, but not move `Request`. Even if the move is superficial, I still think it would make more sense to put this into module_utils.

Eventually, we need to think about lining up the parameters for host, username, password, etc. between the inventory plugin and the modules.